lib/podlet.js

```diff
@@ -818,6 +823,55 @@ export default class PodiumPodlet {
         return this.#view(incoming, data, ...args);
     }
 
+    /**
+     * I'm imaginging this as a helper method to be used in conjunction with render/podiumSend.
+     *
+     * Eg.
+     * app.get("/", (req, res) => {
+     *	res.podiumSend(res.podiumIsolate("<div>content to render</div>"));
+     * });
+     *
+     * This is for 2 reasons.
+     * 1. This gives the podlet developer full control of when to isolate the content without automagic.
+     * 2. This enables podiumSend/render to continue working exactly as they are making this a fully backwards compatible feature.
+     *
+     * We could consider integrating this feature into the render/podiumSend methods but this would require a rework of their method signatures to accommodate.
+     *
+     * We should consider naming of isolate. There may be better, more descriptive names? eg. podlet.wrapContent, podlet.wrapWithShadowDOM etc.
```

Comment:
I think going with something like `.wrapContent` or `.wrapResponseBody` makes sense, as it leaves room for whatever you want and doesn't tie it specifically to the ShadowDOM thing we are looking at now.